plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -42,9 +48,9 @@ export class AwsOrganizationCloudAccountProcessor implements CatalogProcessor {
   static fromConfig(
     config: Config,
     options: {
-      logger: Logger;
+      logger?: Logger;
     },
-  ): AwsOrganizationCloudAccountProcessor;
+  ): Promise<AwsOrganizationCloudAccountProcessor>;
```

Comment:
This one is breaking though. Might want to upgrade this to a `minor` changeset then.

Reply:
Yep good catch